var/spack/repos/builtin/packages/py-rios/package.py

```diff
@@ -27,10 +31,25 @@ class PyRios(PythonPackage):
     version("1.4.11", sha256="b7ae5311f987b32f1afe1fabc16f25586de8d15c17a69405d1950aeada7b748e")
     version("1.4.10", sha256="6324acccc6018f9e06c40370bc366dc459890e8c09d26e0ebd245f6fd46dad71")
 
-    variant("parallel", default=True, description="Enables the parallel processing module")
+    # In 1.4.x, parallel processing was an extra add-on
+    variant(
+        "parallel",
+        default=True,
+        when="@1.4.16:1.4.99",
```

Comment:
You could simply make the range `@1.4.16:1.4` since version ranges -- even as a `when` argument -- are inclusive (see https://spack.readthedocs.io/en/latest/packaging_guide.html#version-ranges).